redaxo/src/addons/be_style/plugins/customizer/assets/js/main.js

```diff
@@ -1,18 +1,15 @@
-$(document).on('pjax:end',   function(){
-    Customizer.init();
-});
-$(document).ready(function(){
-    Customizer.init();
+$(document).on('rex:init', function (event, container) {
```

Comment:
ggf. sollten wir dann das ganze in anlehnung an jquery auch `ready` nennen, d.h. `ready.rex`